CMakeLists.txt

```diff
@@ -12,6 +12,11 @@ include(Macros)
 option(BUILD_SHARED_LIBS "shared/static libs" ON) 
 option(BUILD_TESTS "tests?" OFF)
 
+# used by test-dladdr
+if(BUILD_SHARED_LIBS)
+    add_definitions(-DDLFCN_WIN32_SHARED)
+endif()
```

Comment:
This should not be necessary anymore as this is exported in https://github.com/dlfcn-win32/dlfcn-win32/pull/72/files#diff-148715d6ea0c0ea0a346af3f6bd610d010d490eca35ac6a9b408748f7ca9e3f4R15, right?